packages/admin/cms-admin/src/dam/FileForm/CropSettingsFields.tsx

```diff
@@ -36,32 +37,43 @@ export function CropSettingsFields({ disabled }: Props): JSX.Element {
         <Container>
             <FormSection title={<FormattedMessage id="comet.dam.file.cropSettings.sectionTitle" defaultMessage="Crop/Focus settings" />}>
                 <FieldContainer>
```

Comment:
I think this `FieldContainer` requires the `fullWidth` prop, as the text sometimes breaks into a new line too early. 

```suggestion
                <FieldContainer fullWidth>
```

Reply:
f5bb73d58910d84445298762417e5464b92e0bc9